tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    int buffer_size = 16;
```

Comment:
yeah, read about that solution, too. But found this a quicker fix, by removing `const`. Anyway, there are more of these to fix, so I can adapt to the `enum` - on the other hand its a test not shipped core code of RIOT